src/codeu/chat/common/UserCollection.java

```diff
@@ -0,0 +1,16 @@
+package codeu.chat.common;
+
+import java.util.ArrayList;
+
+/**
+ * This class is simply to hold all of the users so it is easy to save them to a text file later
+ * Created by himonal on 6/13/17.
+ */
+public class UserCollection {
+    public User[] users;
+
+    public UserCollection(ArrayList<User> currentUsers) {
+        users = new User[currentUsers.size()];
+        users = currentUsers.toArray(users);
```

Comment:
Instead of creating these three classes `UserCollection`, `MessageCollection`, and `ConversationCollection`, you should just use the built-in toArray() method of ArrayList. It doesn't require you to declare an array first, or supply one as a parameter.
`User[] userArray = currentUsers.toArray();`
Should be adequate, and then you won't need the extra classes